manim/animation/creation.py

```diff
@@ -7,16 +7,40 @@
 
     class CreationModule(Scene):
         def construct(self):
-            texts = [Text('manim'), Text('manim')]
-            texts[0].shift(LEFT * 2 + UP)
-            texts[1].shift(RIGHT * 2 + UP)
+            s1 = Square()
+            s2 = Square().next_to(s1, RIGHT, buff=1.9)
+            s3 = Square().next_to(s2, RIGHT, buff=1.9)
+            s4 = Square().next_to(s3, RIGHT, buff=1.9)
```

Comment:
Comment (non-blocking): I believe there is a better way of doing this by using `Mobject.arrange`, but I've never used it so I don't know if it actually works...

Reply:
Uhh, nice!
I will have a look at this.
As nobody will ever see this code, I guess this won't be too important.

Reply:
Ok, I've actually changed it now.
It works like this:
```python
            class Example(Scene):
                def construct(self):
                    s1 = Square()
                    s2 = Square()
                    s3 = Square()
                    s4 = Square()
                    x = VGroup(s1, s2, s3, s4).set_x(0).arrange(buff=1.0)
                    self.add(x)
```